src/components/stickyNavBar.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const StickyNavBar = () => {
+
+  return (
+    <>
+      <style>
```

Comment:
For the sake of consistency and organization, can you put this in a `stickyNavBar.css` file